### PR TITLE
Troubleshooting content and some minor improvements

### PR DESCRIPTION
**Pull request by aetter**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/troubleshoot/openid-connect.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+---
+
+## Set log level to debug
+
+To help troubleshoot OpenID Connect, set the log level to `debug` on Elasticsearch. Add the following lines in `config/log4j2.properties` and restart the node:
+
+```
+logger.opendistro_security.name = com.amazon.dlic.auth.http.jwt
+logger.opendistro_security.level = trace
+```
```
Comment (aetter) [inline docs/troubleshoot/openid-connect.md]: Hey @hardik-k-shah, I could use some help on the correct strings for these logging settings. I wasn't able to dig them up in code.

File: docs/troubleshoot/saml.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+
+## Inspect the SAML response
+
+If you are not sure what the SAML response of your IdP contains and where it places the username and roles, you can enable debug mode in the `log4j2.properties`:
+
+```
+logger.token.name = com.amazon.dlic.auth.http.saml.Token
+logger.token.level = debug
+```
```
Comment (aetter) [inline docs/troubleshoot/saml.md]: Hey @hardik-k-shah, I could use some help on the correct strings for these logging settings. I wasn't able to dig them up in code.

**Comment by aetter** [comment]
@hardik-k-shah
@shivdoshi
@NihalHarish
@viswasou

File: docs/troubleshoot/saml.md (hunk 2 above)
Comment (prashar) [reply docs/troubleshoot/saml.md]: Looks good.

File: docs/troubleshoot/openid-connect.md (hunk 1 above)
Comment (prashar) [reply docs/troubleshoot/openid-connect.md]: Looks good.

**Pull request merged**